cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -719,6 +720,36 @@ __device__ inline dictionary32 const column_device_view::element<dictionary32>(
   return dictionary32{d_children[0].element<int32_t>(index)};
 }
 
+// TODO add docs
+template <>
+__device__ inline numeric::decimal32 const column_device_view::element<numeric::decimal32>(
+  size_type element_index) const noexcept
+{
+  using namespace numeric;
+  if (_type.scale().has_value()) {
```

Comment:
Are we allowing a columns type to be `DECIMAL` and _not_ have a `scale`?

Reply:
If we want the existing unit tests to work - then yes. I was going to bring this up in today's stand up.

Reply:
To elaborate, the use case that requires it is:
```cpp
fixed_width_column_wrapper<numeric::decimal32>{...} // this doesn't set the scale
```
vs
```cpp
`fixed_point_column_wrapper<int32_t>{..., numeric::scale_type{-2}};`
```

Reply:
The decision in the stand was to hardcode `data_type::_scale` for `fixed_width_column_wrapper` so that `column`s for `DECIMAL32` and `DECIMAL64` always have a `scale` set. Will close comment when changes are pushed.